Flex missing when building ebpf module on amazon linux2

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks! -->


**Describe the bug**

The build fails on the latest amazon linux.

falco-crdpc falco * Setting up /usr/src links from host
falco-crdpc falco * Running falco-driver-loader with: driver=bpf, compile=yes, download=yes
falco-crdpc falco * Mounting debugfs
falco-crdpc falco * Found kernel config at /host/boot/config-5.4.91-41.139.amzn2.x86_64
falco-crdpc falco * Using downloaded kernel sources for kernel version 5.4.91...
falco-crdpc falco * Downloading http://mirrors.edge.kernel.org/pub/linux/kernel/v5.x/linux-5.4.91.tar.gz
falco-crdpc falco * Extracting kernel sources
falco-crdpc falco * Configuring kernel
falco-crdpc falco /bin/sh: 1: flex: not found
falco-crdpc falco make[1]: *** [scripts/Makefile.host:9: scripts/kconfig/lexer.lex.c] Error 127
falco-crdpc falco make: *** [Makefile:590: olddefconfig] Error 2
falco-crdpc falco /bin/sh: 1: flex: not found
falco-crdpc falco make[2]: *** [scripts/Makefile.host:9: scripts/kconfig/lexer.lex.c] Error 127
falco-crdpc falco make[1]: *** [Makefile:590: syncconfig] Error 2
falco-crdpc falco Makefile:678: include/config/auto.conf.cmd: No such file or directory
falco-crdpc falco make: *** [Makefile:696: include/config/auto.conf.cmd] Error 2

**How to reproduce it**

Build the ebpf module from source.

**Expected behaviour**

<!-- A clear and concise description of what you expected to happen. -->
The module should build successfully

**Screenshots**


<!-- If applicable, add screenshots to help explain your problem. -->

Amazon Linux 2 / EKS 1.19

<!-- Please complete the following info. -->

- Falco version:
<!-- Use "falco --version". -->

Falco version 0.26.2 and 0.27.0

- System info:
<!-- Falco has a built-in support command you can use  "falco --support | jq .system_info" -->
5.4.91-41.139.amzn2.x86_64

- Cloud provider or hardware configuration:
- OS:
<!-- Eg., output of "cat /etc/os-release". -->
Amazon Linux2

- Kernel:
<!-- Eg., output of "uname -a". -->
5.4.91-41.139.amzn2.x86_64

- Installation method:
<!-- Eg., Kubernetes, RPM, DEB, from source? -->
Chart with building module from source

**Additional context**

<!-- Add any other context about the problem here. -->

--- a/docker/falco/Dockerfile
+++ b/docker/falco/Dockerfile
@@ -18,10 +18,12 @@ RUN apt-get update \
 	&& apt-get install -y --no-install-recommends \
 	bash-completion \
 	bc \
+	bison \
 	clang-7 \
 	ca-certificates \
 	curl \
 	dkms \
+	flex \
 	gnupg2 \
 	gcc \
 	jq \

